database/models/users.js

```diff
@@ -46,10 +42,6 @@ module.exports = (sequelize, DataTypes) => {
           isEmail: {
             args: true,
             msg: 'Please enter a valid email'
```

Comment:
The isUnique description has been removed. Why?

Reply:
It has been removed because it's not a sequelize method. :dancer: